src/libs/Permissions.ts

```diff
@@ -50,8 +50,11 @@ function canUseCustomRules(betas: OnyxEntry<Beta[]>): boolean {
     return !!betas?.includes(CONST.BETAS.CUSTOM_RULES) || canUseAllBetas(betas);
 }
 
+/**
+ * The feature is released so hardcoding the value to true while we clean up the beta from the code.
+ */
 function canUseTableReportView(betas: OnyxEntry<Beta[]>): boolean {
-    return !!betas?.includes(CONST.BETAS.TABLE_REPORT_VIEW) || canUseAllBetas(betas);
+    return true || !!betas?.includes(CONST.BETAS.TABLE_REPORT_VIEW) || canUseAllBetas(betas);
```

Comment:
```suggestion
    return true;
```

Reply:
Then I have to remove the param and TS will scream from the usages